pkgs/development/tools/swc/default.nix

```diff
@@ -5,15 +5,18 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "swc";
-  version = "0.91.19";
+  version = "0.91.67";
+
+  # swc depends on nightly features
+  RUSTC_BOOTSTRAP = 1;
```

Comment:
```suggestion
env = {
  RUSTC_BOOTSTRAP = 1;
  };
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/252534/commits/15b50c89131018ef7c4cdd94a0edc115bf21ea62